I/ITK/build_tarballs.jl

```diff
@@ -0,0 +1,123 @@
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "ITK"
+version = v"5.3.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/InsightSoftwareConsortium/ITK.git", "1fc47c7bec4ee133318c1892b7b745763a17d411")
+]
+#last build failed on bildkite adding comment to build again
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ITK*
+mkdir build/
+cmake -B build -S . \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DBUILD_SHARED_LIBS:BOOL=ON \
+    -DITK_USE_SYSTEM_EXPAT:BOOL=ON \
+    -DITK_USE_SYSTEM_FFTW:BOOL=ON \
+    -DITK_USE_SYSTEM_HDF5:BOOL=ON \
+    -DITK_USE_SYSTEM_JPEG:BOOL=ON \
+    -DITK_USE_SYSTEM_TIFF:BOOL=ON \
+    -DITK_USE_SYSTEM_PNG:BOOL=ON \
+    -DITK_USE_SYSTEM_EIGEN:BOOL=ON \
+    -DITK_USE_SYSTEM_ZLIB:BOOL=ON \
+    -DQNANHIBIT_VALUE:BOOL=0 \
+    -DQNANHIBIT_VALUE__TRYRUN_OUTPUT:STRING=0 \
+    -DVXL_HAS_SSE2_HARDWARE_SUPPORT:STRING=1 \
+    -DVCL_HAS_LFS:STRING=1 \
+    -DDOUBLE_CONVERSION_CORRECT_DOUBLE_OPERATIONS:STRING=1 \
+    -DHAVE_CLOCK_GETTIME_RUN:STRING=0
+cmake --build build --parallel ${nproc}
+cmake --install build
+install_license ${WORKSPACE}/srcdir/ITK/LICENSE
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Platform("x86_64","windows";),
+    Platform("aarch64","linux"; libc="glibc"),
+    Platform("x86_64", "linux"; libc="glibc"),
+    Platform("armv6l","linux"; call_abi="eabihf",libc="glibc"),
+    Platform("armv7l","linux"; call_abi="eabihf",libc="glibc"),
+    Platform("powerpc64le","linux"; libc="glibc"),
+    Platform("aarch64","linux"; libc="musl"),
+    Platform("armv6l", "linux"; call_abi="eabihf", libc="musl"),
+    Platform("armv7l", "linux"; call_abi="eabihf", libc="musl")
+]
```

Comment:
If it works, it works, but in general I would recommend to start with the *full* list of platforms, then remove any that don't work, ideally with a comment indicating what the problem is.

This way, when new platforms are added in the future, they automatically get picked up. And the comments ensure that anyone who wonders "hey, why is macos not supported" has a place to start off.

So here this could be something like this:
```suggestion
platforms = supported_platforms()
filter!(!Sys.isapple, platforms)  # macOS currently not supported because ...
```
.... which actually would leave in also 32bit Windows, I think; but if that really is not supported, it could also be filtered out.

It also would leave in FreeBSD. If things don't work there, just add something like
```
filter!(!Sys.isfreebsd, platforms)  # FreeBSD currently not supported because ...
```